src/cmake.ts

```diff
@@ -1070,243 +1516,18 @@ export class CMakeTools {
         this.statusMessage = 'Refreshing targets...';
         const generator = this.activeGenerator;
         if (generator && /(Unix|MinGW|NMake) Makefiles|Ninja/.test(generator)) {
+            let parser = new TargetParser();

```

Comment:
When I volunteered to improve parser I asked for help to 'polish' changes.
I sent you Collaboration invite.